area_manager/Area myArea
---
bool answer
